Podfile

```diff
@@ -1,13 +1,14 @@
 source 'https://github.com/CocoaPods/Specs'
 
-platform :ios, '8.0'
+platform :ios, '10.0'
```

Comment:
What's the reason for bumping this? I think we should support as many iOS versions as possible.

Reply:
You’re right there’s probably no reason to bump it. But the podspec target isn’t changed either so I assume user of library won’t be affected by this?

Reply:
Yeah, you're probably right. You can keep it then, as long as Intrepid consumers are not limited by this iOS version.

Reply:
This affects our test application, we should keep it at our minimum supported platform.